frontend/src/metabase-types/analytics/event.ts

```diff
@@ -159,6 +159,22 @@ export type EventsClickedEvent = ValidateEvent<{
   triggered_from: "chart" | "collection";
 }>;
 
+export type MoveQuestionsIntoDashboardViewedEvent = ValidateEvent<{
+  event: "move_questions_into_dashboard_viewed";
+  target_id: null;
+  triggered_from: "move_questions_into_dashboards_modal";
+  duration_ms: null;
+  result: string;
+}>;
+
+export type MoveQuestionsIntoDashboardConfirmedEvent = ValidateEvent<{
+  event: "move_questions_into_dashboard_confirmed";
+  target_id: null;
+  triggered_from: "move_questions_into_dashboards_modal";
+  duration_ms: number;
+  result: string;
+}>;
+
```

Comment:
```suggestion
export type MoveQuestionsIntoDashboardViewedEvent = ValidateEvent<{
  event: "move_questions_into_dashboard_viewed";
  triggered_from: "move_questions_into_dashboards_modal";
  result: string;
}>;

export type MoveQuestionsIntoDashboardConfirmedEvent = ValidateEvent<{
  event: "move_questions_into_dashboard_confirmed";
  triggered_from: "move_questions_into_dashboards_modal";
  duration_ms: number;
  result: string;
}>;

```

Not super familiar with our analytics code, but seems like we don't need a couple of fields.